Mathlib/Topology/Algebra/InfiniteSum/Group.lean

```diff
@@ -126,6 +126,26 @@ theorem hasProd_ite_div_hasProd [DecidableEq β] (hf : HasProd f a) (b : β) :
     rw [Function.update_apply]
   · rw [div_mul_eq_mul_div, one_mul]
 
+/-- A more general version of `Multipliable.congr`. -/
+@[to_additive "A more general version of `Summable.congr`"]
```

Comment:
Let's add a comment to `Multipliable.congr` referencing these.

Reply:
Done. Thanks for the review.